src/scss/components/search/_index.scss

```diff
@@ -1,18 +1,18 @@
-@import (reference) "mixins";
-@import (reference) "./SearchFacade";
-
-// ----- LAYOUT -----
-/*
-    auto    1fr    auto
-  +-------+-------+-------+
-  | lbl   | lbl   | lbl   | auto
-  +-------+-------+-------+
-  | icon  | input | btn   | auto
-  +-------+-------+-------+
-  | extra | extra | extra | auto
-  +-------+-------+-------+
-*/
-
+@import "mixins";
+@import "../config";
+
```

Comment:
You'll want to remove this blank line. I'm pretty sure stylelint will gripe.

Reply:
Added documentation comment.